modules/ROOT/pages/quickstart-functions.adoc

```diff
@@ -40,36 +41,49 @@ $ docker login -u <username> -p <password>
 
 . Many `faas` CLI commands require a `--repository` argument.
 +
-You can set the `FAAS_REPOSITORY` environment variable so that you can omit this parameter when using the CLI:
+You can set the `FAAS_REGISTRY` environment variable so that you can omit this parameter when using the CLI:
 +
 
 [source,terminal]
 ----
-$ export FAAS_REPOSITORY=docker.io/<username>/<repository_name>
+$ export FAAS_REGISTRY=docker.io/<username>
 ----
 
 == Create a Functions project
 
-. Create a project directory:
+. Create a project directory, with a function and runtime:
 +
 
 [source,terminal]
 ----
-$ mkdir fn.example.io
+$ mkdir <directory_name> && faas init <function_name> -l node
```

Comment:
```suggestion
$ mkdir <directory_name> && faas init <directory_name> -l node
```
It is true that the directory name and function name are the same, but I think it's probably better to just use one in this case.